client/app/kubernetes/service/reset/kubernetes-service-reset.controller.js

```diff
@@ -9,11 +10,43 @@ class kubernetesResetCtrl {
     this.ControllerHelper = ControllerHelper;
     this.Kubernetes = Kubernetes;
     this.KUBERNETES = KUBERNETES;
+
+    // other attributes used in view
+    this.availableVersions = null;
+
+    this.model = {
+      version: null,
+    };
+
+    this.loading = {
```

Comment:
You can move this directly to $onInit no ?

Reply:
:-) I like to know what are the attributes of my class by seeing them into my constructor. Like this, no need to search for them into other methods.

Reply:
Ok :)